packages/amplify-graphql-conversation-transformer/src/grapqhl-conversation-transformer.ts

```diff
@@ -114,23 +68,4 @@ export class ConversationTransformer extends TransformerPluginBase {
   prepare = (ctx: TransformerPrepareStepContextProvider): void => {
     this.prepareHandler.prepare(ctx, this.directives);
   };
-
-  validate = (): void => {
-    for (const directive of this.directives) {
-      if (directive.field.type.kind !== 'NamedType' || directive.field.type.name.value !== 'ConversationMessage') {
-        throw new InvalidDirectiveError('@conversation return type must be ConversationMessage');
-      }
-      if (directive.handler && directive.functionName) {
-        throw new InvalidDirectiveError("'functionName' and 'handler' are mutually exclusive");
-      }
-      if (directive.handler) {
-        const eventVersion = semver.coerce(directive.handler.eventVersion);
-        if (eventVersion?.major !== 1) {
-          throw new Error(
-            `Unsupported custom conversation handler. Expected eventVersion to match 1.x, received ${directive.handler.eventVersion}`,
-          );
-        }
-      }
-    }
-  };
```

Comment:
This validation logic was moved to `conversation-field-handler.ts` where existing validation logic lives.